hadoop-hdds/docs/content/_index.zh.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Overview
+title: 概述
```

Comment:
There are still some "architecture diagram" and its description part are not localized in this page.

Reply:
do you mean this image ?

![](https://tva1.sinaimg.cn/large/006tNbRwly1ga0lf0cop5j30q40b177b.jpg)

Reply:
That's on the concept page which is currently not localized yet. Let's do that in the follow up JIRAs.